laborator/content/reprezentare-numere/README.md

```diff
@@ -241,77 +264,82 @@ Pentru un șir de caractere dat, să se afișeze:
 >    Address of a: 0x564c364482a1
->
->Pentru șirul de caractere “ababababacccbacbacbacbacbabc”:
->
+>```
+> Pentru șirul de caractere “ababababacccbacbacbacbacbabc”:
+>```
 >    length = 28
 >    Address of b: 0x563f0da6f2a1
 >    Address of a: 0x563f0da6f2a2
 >    Address of c: 0x563f0da6f2a9
->   Address of a: 0x563f0da6f2b0
+>    Address of a: 0x563f0da6f2b0
 >    Address of b: 0x563f0da6f2b2
 >    Address of b: 0x563f0da6f2b5
 >    Address of c: 0x563f0da6f2b7
 >    Address of a: 0x563f0da6f2b9
->
+>```
 >**Adresele de mai sus sunt orientative!**
 
-### 3. Oglindirea unui șir de caractere
+### **3. Oglindirea unui șir de caractere**
 
 Veți rezolva exercițiul plecând de la fișierul **mirror.c** aflat în directorul **3-mirror**.
 
 Folosind operații cu pointeri implementați un program în C care inversează un șir de caractere. Funcția `mirror` trebuie să realizeze rotirea datelor din șirul de caractere **in-place** (la ieșirea din funcție, șirul dat ca intrare va conține șirul inversat).
 
->**NOTE**: Nu accesați elementele șirului sub forma s[i]. 
+>**NOTE**: Nu accesați elementele șirului sub forma `s[i]`.
 
->**TIP**: 
+>**TIP**:
+> ```
 >    mirror(“AnaAreMere”) = “ereMerAanA”
 >
 >    mirror(“asdfghjl”) = “ljhgfdsa”
 >
 >    mirror(“qwerty”) = “ytrewq”
+> ```
 
-### 4. Rotații
+### **4. Rotații**
 
 Veți rezolva exercițiul plecând de la fișierul rotations.c aflat în directorul 4-rotations.
 
-Implementați în C rotație stânga și rotație dreapta pentru numere întregi pe 32 de biți. 
+Implementați în C rotație stânga și rotație dreapta pentru numere întregi pe 32 de biți.
 
 >**TIP**: Operația de rotire (numită și deplasare circulară) este similară cu cea de deplasare, singura diferență fiind aceea că spațiul gol generat de deplasare este înlocuit cu bitul eliminat.
 >
 > Exemplu rotație **stânga** cu **un** bit:
 >
 >![Rotație logică la stânga](images/left-logical-rotation.png)
 
->**NOTE**: 
->    rotate_left(0x80000000, 1) = 1
+>**NOTE**:
+> ```
+>    rotate_left(0x80000000, 1)   = 1
 >    rotate_right(0x00000001, 16) = 65536
+> ```
 
-### 5. Par binar și impar hexa
+### **5. Par binar și impar hexa**
 
 Veți rezolva exercițiul plecând de la fișierul odd_even.c aflat în directorul 5-odd_even.
 
 Să se parcurgă un tablou de întregi pe 32 de biți folosind operații cu pointeri și să se afișeze în binar numerele pare și în hexazecimal numerele impare.
 
->**NOTE**: În rezolvare folosiți operații pe biți oriunde este posibil! 
+>**NOTE**: În rezolvare folosiți operații pe biți oriunde este posibil!
 
->**NOTE**: Pentru vectorul: 214, 71, 84, 134, 86, programul va afișa:
->
+>**NOTE**: Pentru vectorul [214, 71, 84, 134, 86], programul va afișa:
+> ```
 >    0b11010110
 >    0x00000047
 >    0b01010100
 >    0b10000110
 >    0b01010110
+> ```
 
 ## Soluții
 
-Soluțiile pentru exerciții sunt disponibile [aici](https://elf.cs.pub.ro/asm/res/laboratoare/lab-01-sol.zip).
+Soluțiile pentru exerciții sunt disponibile în directorul `./solution/`
```

Comment:
I would honestly remove this mention altogether. It's obvious what the `solution/` folder contains and writing it down might make students overly prone to consult the solution without trying to solve the exercises on their own first.